shared/macros/10-bash.jinja

```diff
@@ -1572,14 +1572,14 @@ Ensures that given mount point is in :code:`/etc/fstab`.
 mount_point_match_regexp="$(printf "[[:space:]]%s[[:space:]]" {{{ mount_point }}})"
 
 # If the mount point is not in /etc/fstab, get previous mount options from /etc/mtab
-if [ "$(grep -c "$mount_point_match_regexp" /etc/fstab)" -eq 0 ]; then
+if ! grep -c "$mount_point_match_regexp" /etc/fstab; then
     # runtime opts without some automatic kernel/userspace-added defaults
     previous_mount_opts=$(grep "$mount_point_match_regexp" /etc/mtab | head -1 |  awk '{print $4}' \
                 | sed -E "s/(rw|defaults|seclabel|{{{ mount_opt }}})(,|$)//g;s/,$//")
     [ "$previous_mount_opts" ] && previous_mount_opts+=","
     echo "{{{ fs_spec }}} {{{ mount_point }}} {{{ type }}} defaults,${previous_mount_opts}{{{ mount_opt }}} 0 0" >> /etc/fstab
 # If the mount_opt option is not already in the mount point's /etc/fstab entry, add it
-elif [ "$(grep "$mount_point_match_regexp" /etc/fstab | grep -c "{{{ mount_opt }}}")" -eq 0 ]; then
+elif ! grep "$mount_point_match_regexp" /etc/fstab | grep -c "{{{ mount_opt }}}"; then
```

Comment:
Why "-c"?

Reply:
Actually we don't need it anymore. I'll fix it.